enterprise/api_client/lms.py

```diff
@@ -490,6 +490,24 @@ def get_course_certificate(self, course_id, username):
         return self.client.certificates(username).courses(course_id).get()
 
 
+class LMSClient(NoAuthenticationLmsApiClient):
```

Comment:
I think name of this Client class should be LMSHealthClient or something like that, otherwise, this LMS client could be used mistakenly with other endpoints (That will be not a good thing as its a NO Authentication Api client)

Reply:
I have renamed all the client and added `NoAuth` in front of each client to clarify.